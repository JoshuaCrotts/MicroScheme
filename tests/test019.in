(define msg "Joshua Loves Vyley <3")
(display msg)
