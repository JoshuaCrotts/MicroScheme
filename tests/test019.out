Joshua Loves Vyley <3
